README.md

```diff
@@ -76,4 +76,13 @@ comp.run_compression(\
   output_file="out.sql",\
   transactions=True\
 )
+```
+
+Note: since this library uses Jemalloc, you might get an error of the form:
+```
+ImportError: /[LONG_PATH]/synapse_compress_state.abi3.so: cannot allocate memory in static TLS block
+```
+If this happens then try running the following:
+```
+LD_PATH=/[LONG_PATH]/synapse_compress_state.abi3.so ./my_python_script
```

Comment:
I wonder if we should instead disable jemalloc when building pyo3? We should be able to do that via feature flags.

Reply:
What was the reasoning for using jemalloc in the first place?

Reply:
It reduces memory usage quite a bit, basically. However, we recommend people run Synapse with jemalloc enabled, which will propagate to rust, and then conflict with the statically linked jemalloc rust used by rust

Reply:
I'm struggling to find a way to disable jemalloc when building pyo3, but I could make it an opt-in feature

But then if you want to use it you'd have to do `cargo build --features jemalloc` which seems to just be adding an inconvenience (especially as the pyo3 thing would only become an issue if it was being called from synapse)

Reply:
have added it as a default feature and can be disabled with:
```
$ maturin develop --cargo-extra-args="--no-default-features"
```